src/Protocol/Diaspora.php

```diff
@@ -2785,15 +2785,15 @@ private static function itemRetraction($importer, $contact, $data)
 
 		while ($item = dba::fetch($r)) {
 			// Fetch the parent item
-			$parent = dba::selectFirst('item', ['author-link', 'origin'], ['id' => $item["parent"]]);
+			$parent = dba::selectFirst('item', ['author-link'], ['id' => $item["parent"]]);
 
 			// Only delete it if the parent author really fits
 			if (!link_compare($parent["author-link"], $contact["url"]) && !link_compare($item["author-link"], $contact["url"])) {
 				logger("Thread author ".$parent["author-link"]." and item author ".$item["author-link"]." don't fit to expected contact ".$contact["url"], LOGGER_DEBUG);
 				continue;
 			}
 
-			Item::deleteById($item["id"]);
+			Item::delete(['id' => $item["id"]]);
```

Comment:
Code standards: Please use a consistent quoting style in the same line at least.